MarcosMdo/subtunes
# Adds a Navigation to the boilerplate

## Describe your changes
Adds a navigation bar
Adds proper page update when choosing "playlist" tab when on subtune view
updates fake user name and pfp
adds theme coloring, display of subtunes

tbh viewsubtunes and viewplaylists are not really needed anymore but no big deal
## Issue Ticket title and link
https://www.notion.so/marcos-mdob/Boiler-Plate-navigation-4dcbb06967a1485598e5ddc8689d44e3?pvs=4
## Test Steps
check out
localhost:3000
login to spotify
click boilerplate home button

check if subtunes load
check if playlists load

check if Library -> Subtune redirects to the subtune tab when on the playlist tab
check if Library -> Playlist redirects to the playlist tab when on the subtune tab


## Checklist before requesting a review
- [x] I have listed testing steps above
